src/Passes/examples/RandomWalkPE2/QSharpVersion/Example.qs

```diff
@@ -0,0 +1,67 @@
+namespace Microsoft.Quantum.Qir.Emission {
```

Comment:
Do we really need all three of these variants of RWPE? I think only 1 and 3 are generally relevant, this one is overly specific to just floating point ranges and rotation units without changing anything about the format or flow compared to version 3.

Reply:
Until we have more examples, I think it is good to have some variations of the same example. The main reason for this is that different ways of constructing Q# code results in different QIRs which will expose different issues. I cannot remember whether it was example 1 or 2, but one of them revealed a segfault the other did not. Hence, I would leave it for now with an expectation that in time we will only need a single version of random walk phase estimation.

Reply:
Hmmm... I see your point: It is basically the exact same code except that you have manually const folded. I revert my standpoint and have deleted 2nd example.